src/main/ruby/core/rbconfig.rb

```diff
@@ -144,10 +144,11 @@ module RbConfig
     mkconfig['topdir'] = '$(archdir)'
   end
 
-  clang          = 'clang'
-  opt            = 'opt'
+  CLANG          = 'clang'
+  OPT            = 'opt'
```

Comment:
Why are they constants now? This means they are exported to the user as RbConfig::CLANG and RbConfig::OPT.

Reply:
> Why are they constants now?

...

> This means they are exported to the user as RbConfig::CLANG and RbConfig::OPT

That's your answer! You can see them used in `mkmf.rb`.

Reply:
I missed it somehow, thanks for the pointer, makes sense.